### DISCUSSION TITLE
Setup of package 'dashboard' at sigenergy/dashboard.yaml, line 1 failed

### DISCUSSION THREAD

**Discussion by Wictorssons** [TypQxQ/Sigenergy-Home-Assistant-Integration#4] [General]
Hi!

Thanks for your work on this integration!

Not sure if it affects anything but after a restart, I get the following error:

-----
Logger: homeassistant.config
Source: config.py:950
First occurred: 14:57:40 (2 occurrences)
Last logged: 15:13:03

Setup of package 'dashboard' at sigenergy/dashboard.yaml, line 1 failed: Invalid package definition 'dashboard': expected dict for dictionary value @ data['title']. Package will not be initialized
-----

It is not clear to me what it means. Any idea what causes this?
The dashboard and its pages display correctly as far as I can tell.

Regards
Martin

**Comment by TypQxQ** [comment]
The code inside the dashboard should be copied to a new dashboard.
I made a short instruction here:
https://github.com/TypQxQ/Sigenergy-Home-Assistant-Integration/wiki/2.-How%E2%80%90to%E2%80%90install#dashboard

Hope this helps 😄 

Regards,
Andrei

**Reply by Wictorssons** [reply]
Now I understand my problem. I had done what you propose and had created a new dashboard but I still had your file, dashboard.yaml, in the integration's folder. So it tried to use that file somehow. After removing that file, the error is gone! Thanks for the pointers!
